src/components/plugin/video.tsx

```diff
@@ -39,12 +39,18 @@ export default class Video extends React.Component<IProps> {
   render () {
     const {word, videoUrl, videoCaption, videoAltText, closedCaptionsUrl, disableReadAloud} = this.props;
     const {lang} = this.context;
+
     return (
       <div className={css.videoContainer}>
       {
-        closedCaptionsUrl ?
+        closedCaptionsUrl || this.translatedClosedCaptionsUrl ?
         <video src={videoUrl} title={videoAltText} crossOrigin={"anonymous"} controls={true}>
-          <track kind="subtitles" srcLang={lang} src={this.translatedClosedCaptionsUrl} default={true}/>
+          <track
+            kind="subtitles"
+            srcLang={lang}
+            src={lang === "en" ? closedCaptionsUrl : this.translatedClosedCaptionsUrl}
```

Comment:
There is a couple of issues here:

1. I think line 46 should be on line 48 instead - right now the video is only shown if there is a closed caption url
2. You should make a const out of this before line 43 and then use it on line 48 and here.  Otherwise there is a chance that there could be a translatedClosedCaptionsUrl but not a closedCaptionsUrl and it would pick the wrong one here.

Something like (with ... for gaps in lines)

```
const trackSrc = lang === "en" ? closedCaptionsUrl : this.translatedClosedCaptionsUrl
...
trackSrc ? <track ...
....
src={trackSrc}
```

Reply:
Actually I think it should be this to allow a fallback:

`const trackSrc = lang === "en" ? closedCaptionsUrl : (this.translatedClosedCaptionsUrl || closedCaptionsUrl)`